### PR TITLE
Added a Geo-Locate button to the FSM Location form

### PR DESCRIPTION
**Pull request by peaseBenjamin**
On the FSM Location, have a tab for "Geo Location" data, similar the "Partner Assignation" tab, including the Geolocate button. Users expect to have the Geo Location features on the FSM location form, as they do on the Partner Form.

### CODE REVIEW

File: fieldservice/models/fsm_location.py [hunk 1]
```diff
@@ -333,6 +333,10 @@ def action_view_sublocation(self):
                                     'form')]
                 action['res_id'] = sublocation.id
             return action
+
```
Comment (dreispt) [inline fieldservice/models/fsm_location.py]:
```suggestion

```

**Review by brian10048 — APPROVED** [review]
Tested and working as intended

**Review by bodedra — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Comment by dreispt** [comment]
Congratulations @peaseBenjamin ! 🎉

**Pull request merged**